src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -1225,10 +1259,10 @@ export default class CommandControl extends React.Component {
 	/**
 	 * Reset mission planning
 	 * 
-	 * @param {MissionInterface} mission used to access the mission state
-	 * @param {boolean} needConfirmation does the deletion require a confirmation by the opertor?
-	 * @param {boolean} rallyPointMission is the mission a rally point mission?
-	 * @return {Promise<boolean>} did the deletion occur? If (yes) then (true)
+	 * @param {MissionInterface} mission Used to access the mission state
+	 * @param {boolean} needConfirmation Does the deletion require a confirmation by the opertor?
+	 * @param {boolean} rallyPointMission Is the mission a rally point mission?
+	 * @returns {Promise<boolean>} Did the deletion occur? If (yes) then (true)
```

Comment:
Does it return false if the deletion did not occur.
Can we remove the parentheses as well?

Reply:
Yes, are you asking because there is only an example in the comments for the "yes" case?

Reply:
Yes, I think we should be explicit so others know what to expect.